lp/ui/marc.py

```diff
@@ -139,8 +139,9 @@ def extract(record, d={}):
     d['URI_AUTHOR']   = get_http_link_set(d['URI_AUTHOR'])
```

Comment:
I missed this earlier, but you'll want to remove the extra spaces here in front of the =. (PEP8)